en_US/messages.json

```
@@ -78,5 +78,35 @@
     "qualityEstimationMessage": {
       "message": "Enable quality estimation",
       "description":"Checkbox to enable the quality estimation of the translation."
+    },
+    "formtranslationsReady": {
+      "message": "Ready",
+      "description":"Notifies that the form translations is ready to be used."
+    },
+    "formtranslationsTyping": {
+      "message": "Typing",
+      "description":"Notifies the user that typing is occuring in the form translations area."
+    },
+    "formtranslationsTranslationInProgress": {
+      "message": "Translation in progress...",
+      "description":"Notifies the user that the translation of the input text is happening."
+    },
+    "formtranslationsTranslationCompleted": {
+      "message": "Translation completed.",
+      "description":"Notifies the user that the translation of the input text is completed."
+    },
+    "formtranslationsDescription": {
+      "message": "This is Form Translations by Firefox Translations. Type in the form to the left in <em>$navigatorLanguage$</em>  and have the form you selected translated to <em>$pageLanguage$</em>. The form to the right represents the text translated back from <em>$pageLanguage$</em> to <em>$navigatorLanguage$</em> for verification.",
```

Why "Form Translations" uppercase? Is it a feature, is it supposed to be kept in English?
```suggestion
      "message": "This is Form Translations by Firefox Translations. Type in the form to the left in <em>$navigatorLanguage$</em> and have the form you selected translated to <em>$pageLanguage$</em>. The form to the right represents the text translated back from <em>$pageLanguage$</em> to <em>$navigatorLanguage$</em> for verification.",
```

Do you have a screenshot of the feature? I don't understand this explanation (e.g. "form you selected").

---

Sure, you can see it working starting from second 25 in this video: https://www.dropbox.com/s/9ghqfu720zr8r2c/Screen%20Recording%202022-03-03%20at%2011.37.50%20PM%281%29.mov?dl=0

---

I agree is a bit difficult to explain that in few words, but this feature a requirement from the EU grant